### ISSUE TITLE
HI!

### ISSUE THREAD

**Issue by beauhaus** [flopywood/hello-world#1]
Good Luck, Florencia 👍 

https://www.youtube.com/watch?v=Bv_5Zv5c-Ts

This really helped me get a handle on .js.

He's really, really slow...--but then, so am I.

~Beau
@BeauWebDev

P.S.
If I were you, try Udacity.
Sololearn is good. Udacity is better. --Most of it is free. (YAY!)

**Comment by flopywood** [comment]
Hey, look what I found http://eloquentjavascript.net/ it seems very helpful!!!

**Comment by beauhaus** [comment]
How did I miss this? 👍 

THANK YOU THANK YOU! :)

Have a great weekend!

P.S.
I think I found u on twitter

You've tweeted more than I thought was humanly possible.

**Comment by flopywood** [comment]
Wait, WHAT? I don't think I'm on twitter but send me the link and I'll check it out

**Comment by LeoFib** [comment]
hahaha...lol I didn't think that picture looked like you! :)
I thought it was a bit weird.

I was trying to figure out whether she was a linguist (translator).

Anyway... I'm following the wrong person lol.

I did that for years in asia...--Now I'm back in the US to change my
career.--A bit like yourself.

okay here is the nerdy link of the day:
https://www.youtube.com/channel/UCpOIUW62tnJTtpWFABxWZ8g

First, the guy is a Brit... so it's somehow easier to listen to... :)

Second... I don't care much for PHP (read, I don't know what the hell it
is!)... But he has a really easy, clear, slow, gentle way to explain what
JSON is and does... There are four videos explaining it...

Also I mostly download (torrent files of Lynda.com and tuts plus) My one
gripe with FCC is that there aren't videos or audio of the most difficult
parts.

I need lots of examples before I learn whatever the F I'm doing.

Anyway...

Code on! DON"T Give up!

~Beau

CU :)

**Comment by beauhaus** [comment]
ah...
@Florencia  is the twitter name I saw.

**Comment by flopywood** [comment]
hahahahaha you made me laugh. That's definitely not me on twitter. 
I found some interesting things. I don't know if you know the site Sitepoint.com but if you don't you should check it out. Sign up and you will be directed to a page that has lots of free books on different topics. I downloaded some about web design, javascrip and html&css. They are great and simply written to understand the concepts.

**Comment by beauhaus** [comment]
--I've been replying to you from my email... I didn't realize that it's
also viewable on Github. lol
 Yes!

Pluralsight is great!

I've started to follow people in the industry...I was so excited to "connect" now, months later, I feel like I'm stuck in a class that I shouldn't have registered for. It's all completely over my head. lol.

Ugh... such a newb..:( :)

Anyway. Pls lemme know if you have trouble with the FCC questions. I did them months ago. I'm going over them from scratch this week.

Chow~
B
fibonacci(two,zero,one,six)at gmail(dot) com
